test/cmdlineTests.sh

```diff
@@ -465,12 +465,12 @@ SOLTMPDIR=$(mktemp -d)
 
     # This should not fail
     set +e
-    output=$(echo '' | "$SOLC" --ast-json - 2>/dev/null)
```

Comment:
So this was redundant?

Reply:
No, it was and still is doing what it was intended to do - making sure that `solc` does not fail with empty stdin. It was added in #5007. It's not stated in the issue but I think that it might have been happening with other options too and `--ast` might just have been chosen as an example.